lib/ui/src/modules/ui/components/header.js

```diff
@@ -33,44 +33,58 @@ const shortcutIconStyle = {
   cursor: 'pointer',
   display: 'inlineBlock',
   padding: 0,
-  margin: '0 0 0 5px',
+  margin: isMobileDevice ? '0 15px' : '0 0 0 5px',
   backgroundColor: 'inherit',
   outline: 0,
   width: 30,
   flexShrink: 0,
+});
+
+const burgerIconStyle = {
+  ...iconStyle(true),
+  paddingBottom: 2,
 };
 
-const linkStyle = {
+const linkStyle = isMobileDevice => ({
   textDecoration: 'none',
   flexGrow: 1,
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
-  border: '1px solid rgb(193, 193, 193)',
+  border: isMobileDevice ? 'none' : '1px solid rgb(193, 193, 193)',
   borderRadius: 2,
-};
+});
 
-const Header = ({ openShortcutsHelp, name, url }) => (
-  <div style={wrapperStyle}>
-    <a style={linkStyle} href={url} target="_blank" rel="noopener noreferrer">
+const Header = ({ openShortcutsHelp, onBurgerButtonClick, name, url, isMobileDevice }) => (
+  <div style={wrapperStyle(isMobileDevice)}>
+    {isMobileDevice && (
+      <button style={burgerIconStyle} onClick={onBurgerButtonClick}>
+        ☰
+      </button>
+    )}
+    <a style={linkStyle(isMobileDevice)} href={url} target="_blank" rel="noopener noreferrer">
       <h3 style={headingStyle}>{name}</h3>
     </a>
-    <button style={shortcutIconStyle} onClick={openShortcutsHelp}>
+    <button style={iconStyle(isMobileDevice)} onClick={openShortcutsHelp}>
```

Comment:
Looks like this button is useless on mobiles, it shows keyboard shortcuts

Reply:
We can actually make those things clickable and perform the action.. but once the left panel is gone (either by toggling it, or by going full-screen, you can't go back)

Reply:
Note that some people use USB keyboards with mobile devices, so I wouldn't necessarily just remove it because of mobile.